app/controllers/domain_tags_controller.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+class DomainTagsController < ApplicationController
+  before_action :authenticate_user!, only: [:add, :remove, :edit, :update, :destroy]
+  before_action :verify_core, only: [:add, :remove, :edit, :update]
+  before_action :verify_admin, only: [:destroy]
+  before_action :set_domain_tag, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @tags = if params[:filter].present?
+              DomainTag.where("name LIKE '%#{params[:filter]}%'")
```

Comment:
Is this a possible SQL injection vulnerability?

Reply:
If it is then we have problems, because the rest of the app uses this format too. I don't *think* it is, but I'll check when I can.

Reply:
Looks like you're right - I can get it to throw a syntax error, but can't get it to do anything malicious. We probably want to go through those other cases and use [`sanitize_sql_like `](//stackoverflow.com/a/32165883/1849664) to prevent some perf issues (https://githubengineering.com/like-injection/).